PanRangeSlider/Common/RangeSlider.cs

```diff
@@ -546,11 +605,34 @@ void OnLayoutPropertyChanged()
         var trackRadius = (float)GetDoubleOrDefault(TrackRadius, trackSize / 2);
         var lowerThumbSize = GetDoubleOrDefault(LowerThumbSize, ThumbSize);
         var upperThumbSize = GetDoubleOrDefault(UpperThumbSize, ThumbSize);
-        Track.StrokeShape = CreateRoundRectangleShape(trackRadius);
-        TrackHighlight.StrokeShape = CreateRoundRectangleShape(trackRadius);
-        LowerThumb.StrokeShape = CreateRoundRectangleShape((float)GetDoubleOrDefault(GetDoubleOrDefault(LowerThumbRadius, ThumbRadius), lowerThumbSize / 2));
-        UpperThumb.StrokeShape = CreateRoundRectangleShape((float)GetDoubleOrDefault(GetDoubleOrDefault(UpperThumbRadius, ThumbRadius), upperThumbSize / 2));
```

Comment:
why did you get rid of CreateRoundRectangleShape usage? now this method is not used